xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-script/src/main/java/org/xwiki/user/script/GroupScriptService.java

```diff
@@ -147,16 +147,16 @@
     }
 
     /**
-     * Checks if a group (candidate) can be added to the members of another group (target).<br />
+     * Checks if a group (candidate) can be added to the members of another group (target).<br>
      * A group (candidate) can be added to the members of another group (target) if the target is not, transitively,
-     * already part of the members of candidate.<br />
-     * A group cannot be added to its own members.<br />
-     * A group (candidate) cannot be added to the members of a group (target) if candidate is already one of the direct
-     * members of target.<br />
+     * already part of the members of the candidate.<br>
```

Comment:
that don't fix the issue at all. Here the problem is about `<` and `>` which are not encoded.